package.json

```diff
@@ -25,7 +25,7 @@
     "master": "node_modules/.bin/babel-node server.js --master",
     "worker": "node_modules/.bin/babel-node server.js --worker",
     "redis": "redis-server",
-    "test": "TEST=1 node_modules/.bin/istanbul cover node_modules/.bin/_mocha -- $(find tests -name '*.test.js') --compilers js:babel-core/register",
+    "test": "TEST=1 ./node_modules/.bin/babel-node ./node_modules/.bin/istanbul cover node_modules/.bin/_mocha -- --compilers js:babel-core/register --timeout 5000 $(find tests -name '*.test.js') ",
```

Comment:
as you are using `babel-node`, does it necessary to put `--compilers js:babel-core/register`?